packages/grafana-ui/src/components/ValueMappingsEditor/ValueMappingsEditor.story.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { ValueMappingsEditor } from './ValueMappingsEditor';
+
+const ValueMappingsEditorStories = storiesOf('UI/ValueMappingsEditor', module);
+
+ValueMappingsEditorStories.add('default', () => {
+  return (
+    <ValueMappingsEditor
+      valueMappings={[]}
+      onChange={() => {
+        action('Mapping changed');
```

Comment:
This won't work. What `action`does, is it returns a handler funtion: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/storybook__addon-actions/index.d.ts#L17

So in this case, what you would like to do is either:

`onChange={action('MappingChanged')}`

or

`onChange={(mappings) => action('MappingChanged')(mappings)}`

What I would expect from using actions addon is to see what are the arguments passed to the handler. So short er version os above is enough.

Reply:
Can't get any of those solutions to work? Simple console.log works though